src/message.rs

```diff
@@ -682,10 +599,6 @@ impl Message {
     }
 
     pub fn is_setupmessage(&self) -> bool {
-        if self.viewtype != Viewtype::File {
-            return false;
-        }
```

Comment:
is this an unrelated change sneaking in?  is it intentional?

Reply:
I reverted it.

I initially removed it because `is_setupmessage` was used to determine where the message should be moved. Now it is determined entirely by prefetch headers, not the `Message`.

I can probably be removed or turned into debug assert, because it is there since the C version to guarantee what `deltachat.h` promises: "For setup messages, dc_msg_get_viewtype() returns DC_MSG_FILE.". But it's not related to this PR.